articles/getting-started/install-configure-bonsai.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: BonsaiInstallationAndConfiguration
+title: Bonsai Installation and Configuration
+---
+
+# Bonsai Installation and Configuration
```

Comment:
The `title` field in the metadata is used as the level 1 header, so we don't need both as far as I know.

Reply:
true